doc/en/object/general.md

```diff
@@ -32,25 +32,25 @@ Using an object literal - `{}` notation - it is possible to create a
 plain object. This new object [inherits](#object.prototype) from `Object.prototype` and 
 does not have [own properties](#object.hasownproperty) defined.
 
-    var foo = {}; // a new empty object
+    var names = {}; // a new empty object
 
-    // a new object with a 'test' property with value 12
-    var bar = {test: 12}; 
+    // a new object with a 'name' property with value 'Rob'
+    var rob = {name: 'Rob'}; 
 
 ### Accessing Properties
 
 The properties of an object can be accessed in two ways, via either the dot
 notation or the square bracket notation.
     
-    var foo = {name: 'kitten'}
```

Comment:
Haha - this is a great change. I wonder y someone decided on 'kitten' and stuck with foo as the variable name :D